oma-console/src/print.rs

```diff
@@ -116,7 +131,22 @@ fn term_color<D>(input: D, color: Action) -> StyledObject<D> {
         Action::PendingBg => style(input).bg(Color::Blue).bold(),
     }
 }
-
+/// OmaLayer
+/// `OmaLayer` is used for output oma-style logs to `tracing`
```

Comment:
is used to

Reply:
”is used to“ means someone did it regularly in the past, and "is used for" indicates the purpose. There is no grammar issues.